lib/azure/armrest/model/base_model.rb

```diff
@@ -222,6 +216,10 @@ module Sql
       class SqlServer < BaseModel; end
       class SqlDatabase < BaseModel; end
     end
+
+    class ArmrestCollection < Array
```

Comment:
Even though very simple, I think `ArmrestCollection` should be in its own `.rb` file, under the same directory.